src/jit/gentree.h

```diff
@@ -2671,7 +2671,7 @@ struct GenTreeCall final : public GenTree
 
 #ifdef FEATURE_READYTORUN_COMPILER
     // Call target lookup info for method call from a Ready To Run module
-    CORINFO_CONST_LOOKUP gtEntryPoint;
+    CORINFO_LOOKUP gtEntryPoint;
```

Comment:
Ah, this means that FEATURE_READYTORUN_COMPILER is defined for the runtime JITing compiler too? Is that right? If that is the case, the overhead at runtime will be significant.